src/listpack.c

```diff
@@ -2344,6 +2393,46 @@ int listpackTest(int argc, char *argv[], int flags) {
         lpFree(lp);
     }
 
+    TEST("Random pairs with many elements and tuple_len 3") {
+        int count = 5;
+        lp = lpNew(0);
+        listpackEntry *keys = zcalloc(sizeof(listpackEntry) * count);
+        listpackEntry *vals = zcalloc(sizeof(listpackEntry) * count);
+
+        lp = lpAppend(lp, (unsigned char*)"abc", 3);
+        lp = lpAppend(lp, (unsigned char*)"123", 3);
+        lp = lpAppend(lp, (unsigned char*)"xxx", 3);
```

Comment:
maybe add big values no more than 2^48-1 to mimic TTL

Reply:
added